gtk4/sample/examples/column_view_demo.rb

```diff
@@ -0,0 +1,76 @@
+# Standalone demo of ColumnView in gtk4.
+
+require "gtk4"
+
+class DataObject < GLib::Object
+    attr_accessor :fruit, :color
```

Comment:
Could you use 2 spaces indent?

Reply:
ok

Reply:
Go ahead and accept the changes, and add the spaces however you want.